### DISCUSSION TITLE
Terminal compatibility and configuration document ...

### DISCUSSION THREAD

**Discussion by evanpolster** [Textualize/textual#2893] [General]
Hello,

Noob question. Not sure where to start on learning terminal emulation and its settings (or how much it is even necessary).

I have created a simple project in the PyCharm IDE and copied in the` textual/docs/examples/how-to/dom4.*` files into it.

When I run `dom4.py` from inside PyCharm its terminal emulator can decipher Unicode characters (like the icon in the Header widget).

When I run (through the same Python virtual environment) from my Gnome desktop terminal it does not decipher the Unicode character _and_  spits out escape sequences to the terminal's stdout visible at  `Ctrl-C` exit.

I am trying to figure out how to configure my terminal with a textual-compliant setup and I was wondering if you could help or point me to a relevant document or discussion.

--Evan

```
Linux: Rocky 9
Desktop: Gnome 40.4
Window: X11
```

- Gnome terminal session:
```
TERM=xterm-256color
GNOME_TERMINAL_SERVICE=:1.78
GDM_LANG=en_US.UTF-8
LANG=en_US.UTF-8
```

- (Inside) PyCharm terminal session:
```
TERM=xterm-256color
TERMINAL_EMULATOR=JetBrains-JediTerm
GDM_LANG=en_US.UTF-8
LANG=en_US.UTF-8
```

**Comment by TomJGooding** [comment]
Could I just double-check the output if you run the below in your terminal?

```
echo -e '\u2b58'
```

Do you know what font is being used?

**Reply by evanpolster** [reply]
When run that echo command I get a one-character white-bordered box with the hex number 2 B on the top row and 5 8 on the bottom.

I am using Source Code Pro as the font.

**Reply by evanpolster** [reply]
Hey Tom,

  So I changed to use Nerd fonts Source Code Pro and it now displays the Unicode character!

Thank you,

--Evan

**Reply by TomJGooding** [reply]
Glad changing the font worked -  I'm a bit surprised that Source Code Pro doesn't seem to support that glyph!
